### DISCUSSION TITLE
Can I use remote (Gravatar) picture as my profile pic?

### DISCUSSION THREAD

**Discussion by MikiVanousek** [jpanther/congo#309] [Q&A]
[This link](https://s.gravatar.com/avatar/5baa4c8c84120b9219fad64f18c0d28a?s=144) leads to my Gravatar. Is there a way to use that as the source of my profile picture?

I like the idea of having to update it at just one place in case I need to change it.

**Comment by jpanther** [comment]
The short answer to this is no, not currently. The theme uses Hugo Pipes to rescale the profile image and as such it needs to be provided as an asset. I can look to make this an option in a future release though, so that if an external URL is provided, it uses that instead of performing any image processing.

**Comment by esycat** [comment]
It sounds like a convenient option to have.
I stumbled upon the [Paper](https://themes.gohugo.io/themes/hugo-paper/) theme that supports it.

**Comment by wolfspyre** [comment]
eh.. yes and no… like almost everything in technology there’s a trade-off.

here, you’re essentially tying your website’s functionality to the availability of another… which kinda flies in the opposite direction of one of the main benefits of a static site… 

what benefit are you hoping to derive from this?

(i’m not trying to be abrasive here, just curious as to the perceived upside, as I don’t see one)

**Reply by wolfspyre** [reply]
thinking out loud and playing the other side here,

i suppose fetching the image at build time and embedding would solve the dependency issue, but that would lead to a lag of updates between rebuilds
 which likely would go against others expectations of external updates applying immediately..

not a huge fan of the idea personally, but the option of having the image(s) outside the project isn’t inherently bad …. just goes against my personal tastes…. which… well… are just as valid as yours…

so… i can see this being a legitimately beneficial option in some contexts so long as it’s not an exclusive requirement…

**Comment by keval6b** [comment]
+1 on this

Using something like https://hugomods.com/docs/gravatar/ might be helpful but doesn't allow for arbitrary urls to be used.

I feel like if you're setting a gravatar, you are implying that you depend on the other site so I don't see the need to grab the image at build.
